src/components/NewLexemeForm.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import { inject } from 'vue';
+import { Button as WikitButton } from '@wmde/wikit-vue-components';
+import { MessagesKey } from '@/plugins/MessagesPlugin/Messages';
+
+const $messages = inject( MessagesKey );
+</script>
+
+<template>
+	<form class="new-lexeme-form">
```

Comment:
I'd say sure, but that means we need to set up some kind of linter to warn us about this using conventions and expecting developers to police each other on it, would be too much overhead IMO.

Reply:
I agree. I don't have the bandwidth to look for a linter for this right now. Let's look into this another time and be more lenient for now.

Reply:
I haven’t found a lint for this yet, unfortunately… I’m not even sure if it would be in eslint or stylelint. Or in both? eslint checking that there is a `class="foo-bar"` attribute matching the file name `FooBar.vue`, and stylelint checking that there is a `.foo-bar` root selector also matching the file name…